bigtable-client-core-parent/bigtable-client-core/pom.xml

```diff
@@ -260,6 +260,14 @@ this is not used directly, but upgrading due to transitive vulnerabilities in ol
       </resource>
     </resources>
     <plugins>
+      <plugin>
+        <!-- skip for bigtable-client-core to allow for upcoming refactoring-->
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>clirr-maven-plugin</artifactId>
+        <configuration>
+          <skip>true</skip>
+        </configuration>
```

Comment:
Not for this PR, but I think it would be really nice to write a maven plugin that generates an [ignoredDifferencesFile](https://www.mojohaus.org/clirr-maven-plugin/check-mojo.html#ignoredDifferencesFile) for clirr that contains everything annotated with InternalApi.

But for now, it seems like we are disabling clirr for every module. Why not just put it in the root pom?

Reply:
👍 that sounds like a good plan. I'd prefer to have it granularly by each module, so that when we're done with the refactorings, we can simply remove it rather than adding it back in with other exclusions (like for shaded poms). Let me know if you disagree!